src/screens/EventScreens/CreateEventScreen/index.js

```diff
@@ -0,0 +1,74 @@
+import React, {useState} from 'react';
+import { View, Text } from 'react-native';
+import { Input, Button } from '../../../components';
+import { styles } from './style';
+import { THEME_COLORS } from '../../../components/theme';
+
+const CreateEventScreen = ({ navigation }) => {
+    const [eventName, setEventName] = useState('');
+    const [eventDate, setEventDate] = useState('');
+    const [eventLocation, setEventLocation] = useState('');
+    const [eventDescription, setEventDescription] = useState('');
+    function handleCancel() {
+        setEventName("");
+        setEventDate("");
+        setEventLocation("");
+        setEventDescription("");
+        navigation.navigate('EventHome');
+    }
+    return (
+      <View style={styles.container}>
+        <View>
+            <Input
+                value={eventName}
+                placeholder="Event Name"
+                autoFocus={true}
+                height={70}
+                onInput={setEventName}
+                borderColor={THEME_COLORS.DEFAULT_BLUE_PRIMARY}
+                backgroundColor={THEME_COLORS.WHITE}
+            />
+            <Input
```

Comment:
any handling on date format? Should we create a date picker for this? (e.g., a series of dropdowns, for selecting year, month, date, and time), also, we should create one for `eventStartTime` and one for `eventEndTime`